corehq/apps/repeaters/models.py

```diff
@@ -287,6 +298,37 @@ def get_auth(self):
             return HTTPDigestAuth(self.username, self.password)
         return None
 
+    def fire_for_record(self, repeat_record):
+        headers = self.get_headers(repeat_record)
+        auth = self.get_auth()
+        payload = self.get_payload_and_handle_exception(repeat_record)
+        url = self.get_url(repeat_record)
+        try:
+            response = simple_post(payload, url, headers=headers, timeout=POST_TIMEOUT, auth=auth)
+        except (Timeout, ConnectionError) as error:
+            log_repeater_timeout_in_datadog(self.domain)
+            self.handle_response(RequestConnectionError(error), repeat_record)
```

Comment:
Hadn't thought of that, but makes sense. I think to me it sounds like kind of a wash, so all things equal I think I'll just leave it how it is.

Agree we should improve how we log errors (and other things) in a repeat attempt, but I think we'll handle that in another PR (or more likely, other PRs).